### ISSUE TITLE
Bug overriding timeouts?

### ISSUE THREAD

**Issue by ryanwitt** [aerospike/aerospike-client-nodejs#55]
I have a client with the following timeout policy:

```javascript
aerospike.client({
    ...
    policies: { timeout: 10 },
    ...
}).connect(...
```

However, I have one operate command with it's own timeout:

```javascript
client.operate(key, operations, {timeout:10000}, function(err, record, metadata, key) {
...
});
```

Many of these calls are failing with the following error:

```javascript
{ code: 9,
  message: 'Client timeout: timeout=1000 iterations=1 failedNodes=0 failedConns=0',
  func: 'as_command_execute',
  file: 'src/main/aerospike/as_command.c',
  line: 445 }
```

Is there something that I'm missing? Or is there a bug in overriding timeouts?

**Comment by GayathriKaliyamoorthy** [comment]
@ryanwitt 

Thanks for reporting this. We found the issue and will be making an official release very soon. 

Thanks

**Comment by GayathriKaliyamoorthy** [comment]
Hi @ryanwitt 

We fixed this and released it in version, 1.0.39. Could you please inform us if you are still facing the same issue?

Thanks

**Comment by ryanwitt** [comment]
@GayathriKaliyamoorthy thanks! Adding an empty metadata object as the new docs suggest fixes it:

```javascript
client.operate(key, operations, {}, {timeout:10000}, function(err, record, metadata, key) {
...
});
```
